### DISCUSSION TITLE
Zip Files Missing From Packaging Utilities

### DISCUSSION THREAD

**Discussion by jrv912** [CrowdStrike/Cloud-AWS#145] [Q&A] [answered]
This step references a .zip file that we need to unzip locally before running the packager.py script (package-without-binary.zip).

where is this .zip file located? I can't locate it in the repo - do we create it manually?

https://github.com/CrowdStrike/Cloud-AWS/blob/main/systems-manager/documentation/AWS-Systems-Manager-Intro.md#option-b---creating-a-package-without-the-installer

**Comment by jshcodes** [comment, marked as answer]
Hi @jrv912 - 

You're correct, these files were removed. We will get this page updated.

In the interim, you can access an easier to consume solution that will download the sensor and bundle it for you [here](https://github.com/CrowdStrike/Cloud-AWS/tree/jan2522-state-manager/state-manager/ssm-distributor). 

> Note: This documentation is still in flight, so it has not yet merged to `main`. When it does, this link will need to be updated.
